dev-client/src/screens/SoilScreen/ColorScreen/ColorScreen.tsx

```diff
@@ -85,43 +100,63 @@ export const ColorScreen = (props: SoilPitInputScreenProps) => {
 
   return (
     <SoilPitInputScreenScaffold {...props}>
-      <Column padding="md">
-        <Row alignItems="flex-end">
-          <Row alignItems="center">
-            <Heading variant="h6">{t('soil.color.title')}</Heading>
-            <InfoOverlaySheetButton Header={t('soil.color.title')}>
-              <Paragraph variant="body1">{t('soil.color.info.p1')}</Paragraph>
-              <BulletList
-                data={[1, 2, 3]}
-                renderItem={i => (
-                  <Text variant="body1">{t(`soil.color.info.bullet${i}`)}</Text>
-                )}
-              />
-              <Paragraph variant="body1">{t('soil.color.info.p2')}</Paragraph>
-              <Paragraph variant="body1">{t('soil.color.info.p3')}</Paragraph>
-            </InfoOverlaySheetButton>
+      <SiteRoleContextProvider siteId={props.siteId}>
+        <Column padding="md">
+          <Row alignItems="flex-end">
+            <Row alignItems="center">
+              <Heading variant="h6">{t('soil.color.title')}</Heading>
+              <InfoOverlaySheetButton Header={t('soil.color.title')}>
+                <Paragraph variant="body1">{t('soil.color.info.p1')}</Paragraph>
+                <BulletList
+                  data={[1, 2, 3]}
+                  renderItem={i => (
+                    <Text variant="body1">
+                      {t(`soil.color.info.bullet${i}`)}
+                    </Text>
+                  )}
+                />
+                <Paragraph variant="body1">{t('soil.color.info.p2')}</Paragraph>
+                <Paragraph variant="body1">{t('soil.color.info.p3')}</Paragraph>
+              </InfoOverlaySheetButton>
+            </Row>
+            <Box flex={1} />
+            <RestrictBySiteRole
+              role={[
+                {kind: 'project', role: 'MANAGER'},
```

Comment:
Might be nice to extract this specific combination of parameters to a named constant somewhere, in a dedicated permissions file/folder as noted above.